core/app-config-service/config-release.yaml

```diff
@@ -15,4 +15,25 @@ spec:
     createNamespace: true
   interval: 5m0s
   releaseName: azure-app-config-service
-  targetNamespace: app-config-service
\ No newline at end of file
+  targetNamespace: app-config-service
+  values:
+    autoscaling:
+      enabled: true
+      minReplicas: 2
+      maxReplicas: 5
+    tolerations:
+      - key: "CriticalAddonsOnly"
+        operator: "Equal"
+        value: "true"
+        effect: "NoSchedule"
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.azure.com/mode
+              operator: In
+              values:
+                - 'system'
+    nodeSelector:
```

Comment:
I think we should add in the pod disruption budget here. ?

Reply:
helm chart doesn't have support for the pod disruption budget yet...